Sources/StreamChat/Database/DTOs/AttachmentDTO.swift

```diff
@@ -22,11 +22,11 @@ class AttachmentDTO: NSManagedObject {
     }
 
     /// An attachment local state.
-    @NSManaged private var localStateRaw: String?
+    @NSManaged private var localStateRaw: String
     @NSManaged private var localProgress: Double
     var localState: LocalAttachmentState? {
         get {
-            localStateRaw.flatMap { LocalAttachmentState(rawValue: $0, progress: localProgress) }
+            LocalAttachmentState(rawValue: localStateRaw, progress: localProgress)
```

Comment:
What about `LocalAttachmentState(rawValue: localStateRaw, progress: localProgress) ?? .unknown` so we don't expose an optional?